Duplicati/Server/webroot/ngax/templates/backends/s3.html

```diff
@@ -47,8 +47,7 @@
     <select name="s3_storageclass" id="s3_storageclass" ng-model="$parent.s3_storageclass"
             ng-options="v as k + ' (' + v + ')' for (k, v) in s3_storageclasses | orderBy: k"
             title="{{'The storage class affects the availability and price for a stored file' | translate}}">
-        <option value="" translate translate-params-class="s3_storageclass_custom || ''">Custom storage class
-            ({{class}})
+        <option value="" translate translate-params-class="s3_storageclass_custom || ''">Custom storage class ({{class}})
```

Comment:
This change should be merged with this:
https://github.com/duplicati/duplicati/blob/c955b979fe591b52c040e00ede3b14d2c4e5624d/Localizations/webroot/localization_webroot.pot#L774-L776